Jenkinsfile_CNP

```diff
@@ -19,7 +19,8 @@ def branchesToSync = ['demo', 'ithc', 'perftest']
 
 // Variables to switch pipeline logic and wiring per type of build
 
-def definitionStoreDevelopPr = "PR-575"  // This doesn't change frequently, but when it does, only change this value.
+// TODO: remove TEMPORARY OVERRIDE FOR GlobalSearch R2, reinstate: def definitionStoreDevelopPr = "PR-575"
+def definitionStoreDevelopPr = "PR-1130"  // This doesn't change frequently, but when it does, only change this value.
```

Comment:
Temporary values should be updated prior to merge into `Global_Search_Release2` branch and then removed before that branch is merged for release.